tiled/config.py

```diff
@@ -195,9 +195,13 @@ def construct_build_app_kwargs(
         for ext, media_type in config.get("file_extensions", {}).items():
             serialization_registry.register_alias(ext, media_type)
 
-        for spec, validator_path in config.get("validation", {}).items():
-            validator = import_object(validator_path)
-            validation_registry.register(spec, validator)
+        for item in config.get("specs", []):
```

Comment:
Was this change necessary because you want to be able to support more configuration with specs in the future?

Reply:
Yes, also, just low this comment, notice that validation becomes optional. If the key new `validator` is omitted, then a no-op function is used.

Reply:
That makes sense. Thanks.